exercises/practice/difference-of-squares/difference_of_squares.zig

```diff
@@ -0,0 +1,11 @@
+pub fn square_of_sum(number: isize) isize {
+    @panic("compute the sum of i from 0 to n then square it");
```

Comment:
The panic message contains a description of what to implement, which in itself is very useful. This won't be possible for the more advanced exercises though, in which case you might want to just output something like "Please implement function X".

Reply:
I'll make the change now to this exercise now so that the panic messages are more consistent throughout all of the practice exercises.

Reply:
I'll keep this in mind, when making more complex exercises.